girder/constants.py

```diff
@@ -41,18 +43,22 @@
 TEXT_SCORE_SORT_MAX = 200
 
 # Get the version information
-VERSION = {  # Set defaults in case girder-version.json doesn't exist
-    'git': False,
-    'SHA': None,
-    'shortSHA': None,
-    'apiVersion': None,
-    'date': None
-}
+_gitSha = None
 try:
-    with open(os.path.join(PACKAGE_DIR, 'girder-version.json')) as f:
-        VERSION.update(json.load(f))
-except IOError:
+    # Get the current Git head
+    _gitSha = subprocess.check_output(
+        ['git', 'rev-parse', 'HEAD'],
+        cwd=ROOT_DIR
+    ).decode().strip()
+except subprocess.CalledProcessError:
+    # "gitSha" is None
     pass
+VERSION = {
+    # Set by __init__.py
+    'release': girder.__version__,
+    # TODO: set this
```

Comment:
TODO?

Reply:
That's no longer TODO, but I forgot to remove the comment. Fixed.